src/components/I18n/I18nController.js

```diff
@@ -9,19 +9,19 @@ import { find } from 'lodash';
 
 import localeMessages from 'js/i18n/loadLocaleData';
 import {
-  SUPPORTED_LANGUAGES,
   SUPPORTED_LANG_CODES,
   LANG_COOKIE_NAME,
   LANG_COOKIE_EXPIRES,
   DEFAULT_LANG,
 } from 'js/i18n/constants';
 
-// page_sections
+import I18nDecorator from 'components/I18n/I18nDecorator';
+import SkipToMain from 'components/PageSections/SkipToMain';
 import LanguageSelectBar from 'components/PageSections/LanguageSelectBar';
 import Header from 'components/PageSections/Header';
 import Footer from 'components/PageSections/Footer';
 
-class LanguageWrapper extends Component {
+class I18nController extends Component {
```

Comment:
Thanks for sharing the article! My thinking was what Controller means in a broad sense -- code that acts as a processing layer between data and views. So this I18nController's ONLY responsibility is to determine what language/locale we need to set for the user and to pass the appropriate data to view modules based on that language. So this component shouldn't have anything to do with layout or styles, etc. 

I was even thinking of moving out the children components to a general page layout component (or throwing them in App.js as children), in case we have different Header, Footer, Etc, needs for other city pages and so this controller isn't responsible for page layout.

Hopefully that makes some sense.